### PR TITLE
New Template : Show Etherchannel Summary

### PR DESCRIPTION
**Pull request by alekgozali**
##### ISSUE TYPE

 - New Template Pull Request : Show Etherchannel Summary 

##### COMPONENT

name Template : Show Etherchannel Summary
OS : , cisco_ios 
command : Show etherchannel summary
##### SUMMARY

```

```

### CODE REVIEW

**Comment by alekgozali** [comment]
Added with show etherchannel summary.
i tried to use development scripts, but it gave me this error everytime from development script file itself
    comment.value = f"{space}# {remark_formatted.lstrip()}\n"
am i missing something

**Comment by FragmentedPacket** [comment]
Are you attempting to run it with Python 3.6+ or newer? Can you provide the actual traceback?

If you don't mind cleaning up the PR a bit as well with just the necessary template, index file, and raw/yml files.

**Comment by alekgozali** [comment]
> Are you attempting to run it with Python 3.6+ or newer? Can you provide the actual traceback?
> 
> If you don't mind cleaning up the PR a bit as well with just the necessary template, index file, and raw/yml files.


\ntc_template_offline\ntc-templates>development_scripts.py -yd test/cisco_ios/dir
  File "C:\Users\agozali.LOGICALIS\PycharmProjects\ntc_template_offline\ntc-templates\development_scripts.py", line 111
    comment.value = f"{space}# {remark_formatted.lstrip()}\n"
                                                            ^
SyntaxError: invalid syntax

i'm windows user with python 3.8 version.
sure, let me clean those up.

File: templates/cisco_ios_show_etherchannel_summary.textfsm [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  ^.+(-|LACP|PAgP)\s+[\w\.\/\(\)]+\s+[\w\.\/\(\)]+\s+${INTERFACES}\(${INTERFACES_STATUS}\) -> Continue
+  ^.+(-|LACP|PAgP)\s+[\w\.\/\(\)]+\s+[\w\.\/\(\)]+\s+[\w\.\/\(\)]+\s+${INTERFACES}\(${INTERFACES_STATUS}\) -> Continue
+  ^\s+${INTERFACES}\(${INTERFACES_STATUS}\) -> Continue
+  ^\s+[\w\.\/\(\)]+\s+${INTERFACES}\(${INTERFACES_STATUS}\) -> Continue
+  ^\s+[\w\.\/\(\)]+\s+[\w\.\/\(\)]+\s+${INTERFACES}\(${INTERFACES_STATUS}\) -> Continue
+  ^\s+[\w\.\/\(\)]+\s+[\w\.\/\(\)]+\s+[\w\.\/\(\)]+\s+${INTERFACES}\(${INTERFACES_STATUS}\) -> Continue
+  ^\d+
+  ^\s+
+  ^-+\++
+  ^.+ -> Error
```
Comment (FragmentedPacket) [inline templates/cisco_ios_show_etherchannel_summary.textfsm]:
```suggestion
  ^\s*$$
  ^. -> Error
```
And the `\s+` above can most likely be removed as well.

Once that is changed, we will get this merged

Comment (alekgozali) [reply templates/cisco_ios_show_etherchannel_summary.textfsm]: thank you for your review :) 
it's my honor to be able participate on this code

**Pull request merged**